sim/include/proto/userValue.h

```diff
@@ -65,11 +68,23 @@ requires ProtobufType<T> bool isValOfType(
 }
 
 // Validate if the given val matches one of the listed protobuf types
-template <class... AllowedTypes>
+template <class... Ts>
 // This fold expression validates that each type given is a protobuf type
-requires(proto::ProtobufType<AllowedTypes>&&...) bool isValOfTypes(
+requires(proto::ProtobufType<Ts>&&...) bool isValOfTypes(
     const bento::protos::Value& val) {
-    return (proto::isValOfType<AllowedTypes>(val) || ...);
+    return (proto::isValOfType<Ts>(val) || ...);
+}
+
+template <class T>
+requires ProtobufType<T> bool isTypeOfType(
```

Comment:
`IsProtoTypeOfType`? `IsTypeOfType` sounds like recursive expression.

Reply:
Corrected in 04796f7